Makefile

```diff
@@ -6,6 +6,8 @@ ifeq ($(GOARCH),x86_64)
 	GOARCH=amd64
 else ifeq ($(GOARCH),aarch64)
 	GOARCH=arm64
+else ifeq ($(GOARCH),loongarch64)
```

Comment:
We might want to add an item for release notes? if not I can skip them but I think it could be useful to communicate this.

Reply:
Made additional commit with only short description.
In changelog provided broad description and left link to this PR for details